[2023-01-26 17:53:05,694] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Esperando_el_fichero scheduled__2023-01-01T00:00:00+00:00 [queued]>
[2023-01-26 17:53:05,710] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Esperando_el_fichero scheduled__2023-01-01T00:00:00+00:00 [queued]>
[2023-01-26 17:53:05,710] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 17:53:05,710] {taskinstance.py:1377} INFO - Starting attempt 9 of 11
[2023-01-26 17:53:05,710] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 17:53:05,732] {taskinstance.py:1397} INFO - Executing <Task(FileSensor): Esperando_el_fichero> on 2023-01-01 00:00:00+00:00
[2023-01-26 17:53:05,737] {standard_task_runner.py:52} INFO - Started process 112 to run task
[2023-01-26 17:53:05,741] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Respuesta_Nasa', 'Esperando_el_fichero', 'scheduled__2023-01-01T00:00:00+00:00', '--job-id', '625', '--raw', '--subdir', 'DAGS_FOLDER/00-ETL_spaceX.py', '--cfg-path', '/tmp/tmp9ifs5a3m', '--error-file', '/tmp/tmprpnw65wm']
[2023-01-26 17:53:05,743] {standard_task_runner.py:80} INFO - Job 625: Subtask Esperando_el_fichero
[2023-01-26 17:53:05,821] {task_command.py:371} INFO - Running <TaskInstance: Respuesta_Nasa.Esperando_el_fichero scheduled__2023-01-01T00:00:00+00:00 [running]> on host da6de88268cf
[2023-01-26 17:53:05,933] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Respuesta_Nasa
AIRFLOW_CTX_TASK_ID=Esperando_el_fichero
AIRFLOW_CTX_EXECUTION_DATE=2023-01-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=9
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-01T00:00:00+00:00
[2023-01-26 17:53:05,943] {base.py:68} INFO - Using connection ID 'fs_default' for task execution.
[2023-01-26 17:53:05,943] {filesystem.py:58} INFO - Poking for file /tmp/response_20230101.txt
[2023-01-26 17:53:05,943] {filesystem.py:63} INFO - Found File /tmp/response_20230101.txt last modified: 20230126175304
[2023-01-26 17:53:05,943] {base.py:301} INFO - Success criteria met. Exiting.
[2023-01-26 17:53:05,956] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Respuesta_Nasa, task_id=Esperando_el_fichero, execution_date=20230101T000000, start_date=20230126T175305, end_date=20230126T175305
[2023-01-26 17:53:05,994] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-26 17:53:06,119] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
